tools/ngsderive.wdl

```diff
@@ -153,7 +153,7 @@ task encoding {
 
         ngsderive encoding --verbose \
             -n ~{num_samples} \
-            ~{sep=' ' ngs_files} \
+            ~{sep(",", ngs_files)} \
```

Comment:
Did the separator change in `ngsderive`?

Reply:
No it did not! Good catch.